Makefile

```diff
@@ -962,6 +963,14 @@ release-only: check-xz
 	fi
 
 $(PKG): release-only
+ifneq ($(OSTYPE),darwin)
```

Comment:
A comment just above here explaining why it's restricted to darwin-arm64 might be helpful - I can see some issues with downstream users who might be using this target for custom builds who are surprised that they are now unable to use it. If they want to make it usable on x64 then they can come and contribute here to make it possible to do non-fat packages (or fat on x64 if they want to try).

Something like:
```
# pkg building is currently only supported on a ARM64 macOS host for ease of
# compiling fat-binaries for both macOS architectures.
```